awatif-ui/src/App.ts

```diff
@@ -33,11 +36,16 @@ export function app({ model, parameters, onParameterChange, settings }: App) {
     deformedShape: van.state(settings?.deformedShape ?? false),
     elementResults: van.state(settings?.elementResults ?? "none"),
     nodeResults: van.state(settings?.nodeResults ?? "none"),
+    dynamic: van.state(settings?.dynamic ?? false),
+    dynamicSettings: van.state(
+      settings?.dynamicSettings ?? { time: 1, timeStep: 1 }
+    ),
   };
 
   // update
   Viewer(modelState, settingsState);
   Settings(settingsState);
+  if (settings?.dynamic ?? false) Timeline(modelState, settingsState);
```

Comment:
We can check if there is a dynamic result by inspecting the `analysisResult` state, like so:
```modelState.val.analysisResults.position.size > 1 ```
This way the app is smart enough to show the slider or not